.travis.yml

```diff
@@ -14,13 +17,14 @@ matrix:
   allow_failures:
     - rvm: rbx-3
     - rvm: ruby-head
-sudo: false
 addons:
   apt:
     packages:
     - libpcre3
     - libpcre3-dev
-before_install:
-  - gem install bundler
+env:
+  global:
+  - JRUBY_OPTS='--debug'
```

Comment:
@bklang FYI I think adding `JRUBY_OPTS='--debug'` on this particular PR was needed in order to prevent Coveralls from reporting that coverage was dropping.